diff -up openexr_viewers-2.1.0/exrdisplay/Makefile.am.headers openexr_viewers-2.1.0/exrdisplay/Makefile.am
--- openexr_viewers-2.1.0/exrdisplay/Makefile.am.headers	2014-01-31 10:09:46.514343641 -0600
+++ openexr_viewers-2.1.0/exrdisplay/Makefile.am	2014-01-31 10:11:59.636924846 -0600
@@ -44,7 +44,8 @@ noinst_HEADERS =  \
         scaleImage.h \
         applyCtl.h \
         GlWindow3d.h \
-        GlWindow3d.cpp
+        GlWindow3d.cpp \
+        namespaceAlias.h
 
 EXTRA_DIST = $(exrdisplay_SOURCES)
 
