diff -up openexr_viewers-1.0.2/playexr/ctlToLut.cpp.gccCg openexr_viewers-1.0.2/playexr/ctlToLut.cpp
--- openexr_viewers-1.0.2/playexr/ctlToLut.cpp.gccCg	2007-07-26 04:34:32.000000000 +0200
+++ openexr_viewers-1.0.2/playexr/ctlToLut.cpp	2010-09-17 17:53:19.000000000 +0200
@@ -38,6 +38,7 @@
 //
 //-----------------------------------------------------------------------------
 
+#include <stdio.h>
 #include <ctlToLut.h>
 
 #if HAVE_CTL_INTERPRETER
